### ISSUE TITLE
使用ner few-shot 自定义中文数据集训练的模型，predict预测的时候报错

### ISSUE THREAD

**Issue by choshiho** [zjunlp/DeepKE#539]
**Describe the bug**
使用ner few-shot predict预测的时候报错
<img width="1466" alt="iShot_2024-06-20_18 51 01" src="https://github.com/zjunlp/DeepKE/assets/17508435/e1d3b972-d30c-441a-826b-b46b643c224f">

**config.yaml中已添加 - predict，相关参数配置参考以下代码**：
`cwd: ??? 
use_wandb: False

defaults:
  #- train/conll
  \- predict`

**predict.yaml中相关参数配置参考以下代码：**
`cwd: ??? 

seed: 1

bart_name: "/home/zhifeng.zhao/DeepKE/example/ner/few-shot/bart-base-chinese"
dataset_name: hainan-lianhua
device: cuda

num_epochs: 30
batch_size: 16
learning_rate: 2e-5
warmup_ratio: 0.01
eval_begin_epoch: 16
src_seq_ratio: 0.6 
tgt_max_len: 10
num_beams: 1
length_penalty: 1

use_prompt: True
prompt_len: 10
prompt_dim: 800 

freeze_plm: True
learn_weights: True
notes: ''
save_path: "/home/zhifeng.zhao/DeepKE/example/ner/few-shot/" # 模型保存路径
load_path: "/home/zhifeng.zhao/DeepKE/example/ner/few-shot/hainan-lianhua_5_5e-05/best_model.pth" # 模型加载路径，不能为空
write_path: "/home/zhifeng.zhao/DeepKE/example/ner/few-shot/predict_zzf.txt"`

**标签映射已经配置：**
`MAPPING = {
    'conll2003': {'loc': '<<location>>',
                'per': '<<person>>',
                'org': '<<organization>>',
                'misc': '<<others>>'},
    'mit-movie': mit_movie_mapping,
    'mit-restaurant': mit_restaurant_mapping,
    'cluener2020': cluener2020_mapping,
    'atis': atis_mapping,
    'hainan-lianhua': {
                        'device': '<<device>>',
                        'problem': '<<problem>>',
                        'equipment': '<<equipment>>',
                        'department': '<<department>>',
                        'organization': '<<organization>>'}
}`

**Environment (please complete the following information):**
 - OS: [Ubuntu 22.04]
 - Python Version [Python 3.8.19]

**Screenshots**
> If applicable, add screenshots to help explain your problem.

**Additional context**
> Add any other context about the problem here.

**Comment by njcx-ai** [comment]
您好，非常感谢您对我们工作的关注。由于这个issue问题非常具体，我们相关代码维护同学将尽快排查下给您回复。

**Comment by choshiho** [comment]
> 您好，非常感谢您对我们工作的关注。由于这个issue问题非常具体，我们相关代码维护同学将尽快排查下给您回复。
您好，编辑DeepKE/example/ner/few-shot/conf/predict.yaml，参数修改为learn_weights: False，解决了已经训练好模型的加载问题

**Comment by choshiho** [comment]
> > 您好，非常感谢您对我们工作的关注。由于这个issue问题非常具体，我们相关代码维护同学将尽快排查下给您回复。

您好，编辑DeepKE/example/ner/few-shot/conf/predict.yaml，参数修改为learn_weights: False，解决了已经训练好模型的加载问题
